modules/benchmarks/.factorypath

```diff
@@ -0,0 +1,176 @@
+<factorypath>
```

Comment:
this change of course should be excluded :)

Reply:
changed in [a51932a](https://github.com/dCache/dcache/pull/7532/commits/a51932a9058dcc27955354b86ef00d2561012e0e)